.github/renovate.json5

```diff
@@ -0,0 +1,91 @@
+{
+  "extends": [
+    "config:recommended",
+    ":approveMajorUpdates",
+  ],
+  // submit on Monday midnight (UTC)
+  "schedule": [
+    "after 12am on Monday"
+  ],
+  "minimumReleaseAge": "14 days",
+  "vulnerabilityAlerts": {
+    "labels": [
+      "type:security"
+    ],
+    "minimumReleaseAge": "1 days"
+  },
+  "labels": ["dependencies"],
+  "dependencyDashboardLabels": ["type: process"],
+  "dockerfile": {
+    "enabled": false
+  },
+  "semanticCommits": true,
+  "pip-compile": {
+    "enabled": true,
+    "fileMatch": ["(^|/)requirements\\.in$"],
+    "lockFileMaintenance": {
+      "enabled": true
+    },
+  },
+  // pip-compile handles requirements.txt updates.
+  "pip_requirements": {
```

Comment:
We should not disable this manager here, because there are many `requirements.txt` files that are not generated by pip-compile in this repo.

Reply:
I am (almost) sure that I migrated it from the original. Based on the comment I was under impression that it enforces handling of the .txt file to the `pip-compile`. Am I wrong?

Reply:
BoA **exclusively** uses pip-compile to turn `requirements.in` into `requirements.txt`, so it is safe to disable this manager there. That is not the case for this repository - here we have a mix of "regular" and pip-compile requirements files.
I do not know if these two methods can peacefully coexist, but we certainly should not be disabling pip_requirements in this repo.